napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1487,3 +1495,34 @@ def get_users(self):
                     continue
                 users[username]["sshkeys"].append(py23_compat.text_type(sshkeyvalue))
         return users
+
+    @staticmethod
+    def _get_table_rows(parent_table, table_name, row_name):
```

Comment:
I was under the impression that this function is already defined in https://github.com/napalm-automation/napalm/blob/develop/napalm/nxos/nxos.py#L705, we might want to have it as a helper function and use it in both drivers, rather than duplicating the code.

Reply:
Can transfer these three functions to the NXOSDriverBase class?
Or there are other suggestions ....

Reply:
Correct, this is where I would transfer those functions @VictorPavlushin.

Reply:
Done